### DISCUSSION TITLE
Katex can't render &= ?

### DISCUSSION THREAD

**Discussion by shi-yan** [KaTeX/KaTeX#3786] [Q&A]
The following can't be rendered by katex because of &=

```latex
S(\omega) &= \frac{\alpha g^2}{\omega^5} e^{[ -0.74\bigl\{\frac{\omega U_\omega 19.5}{g}\bigr\}^{\!-4}\,]}\\
&= \frac{\alpha g^2}{\omega^5} \exp\Bigl[ -0.74\Bigl\{\frac{\omega U_\omega 19.5}{g}\Bigr\}^{\!-4}\,\Bigr]
```

Looking at the document, &= seems to be supported by auto-render, but I don't want katex to auto render equations. 

am I missing anything?

Thank you!

**Comment by edemaine** [comment]
Did you mean to wrap that example in `\begin{align*}` ... `\end{align*}` or something else? Without a wrapping environment, `&` has no defined meaning.
